Formula/daemonize.rb

```diff
@@ -18,4 +18,17 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    dummy_script_file = testpath/"script.sh"
+    output_file = testpath/"outputfile.txt"
+    pid_file = testpath/"pidfile.txt"
+    dummy_script_file.write "echo #{version} >> #{output_file}"
```

Comment:
This happens to work on OS X, but relies on some ancient compatibility feature. It's preferable to add a shebang line to the shell script to make it clear to the operating system how to execute the script. You can create a multi-line file like so (it's also a good idea to quote strings in the shell script):

```ruby
dummy_script_file.write <<-EOS.undent
  #!/bin/sh
  echo "#{version}" >> "#{output_file}"
EOS
```

Other than that the code looks good. Many thanks for adding a test!